src/test/kotlin/no/nav/tilleggsstonader/soknad/varsel/DittNavKafkaProducerTest.kt

```diff
@@ -0,0 +1,39 @@
+package no.nav.tilleggsstonader.soknad.varsel
+
+import io.mockk.mockk
+import no.nav.tilleggsstonader.soknad.prosessering.SendNotifikasjonTask
+import org.intellij.lang.annotations.Language
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class DittNavKafkaProducerTest {
+
+    private lateinit var dittNavKafkaProducer: DittNavKafkaProducer
+    private lateinit var sendNotifikasjonTask: SendNotifikasjonTask
+
+    @BeforeEach
+    fun setUp() {
+        dittNavKafkaProducer = mockk(relaxed = true)
+        sendNotifikasjonTask = SendNotifikasjonTask(dittNavKafkaProducer)
+    }
```

Comment:
```suggestion
    private val dittNavKafkaProducer = mockk<DittNavKafkaProducer>(relaxed = true)
    private val sendNotifikasjonTask SendNotifikasjonTask(dittNavKafkaProducer)
```

Samtidig så er dette en test for `DittNavKafkaProducer`, men den er mocket ut?

Reply:
Den er flyttet da det blie ein test for tasken og ikkje produceren.